src/ucs/config/parser.h

```diff
@@ -377,10 +378,17 @@ ucs_config_parser_set_default_values(void *opts, ucs_config_field_t *fields);
 /**
  * Parse INI configuration file with UCX options.
  * 
- * @param path     Path file at this path.
+ * @param path     Parse file at this path.
  * @param override Whether to override, if another file was previously parsed
  */
-ucs_status_t ucs_config_parse_config_file(const char *path, int override);
+void ucs_config_parse_config_file(const char *path, int override);
```

Comment:
maybe remove it from H file and make it static?

Reply:
Can't, it's needed for tests.

Reply:
ok